src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java

```diff
@@ -2477,28 +2477,28 @@ private static ShipComponent createWeapon(final int index) {
       tmp.setEnergyRequirement(1);
     }
     if (index == COMPONENT_PLASMA_CANNON_MK2) {
-      tmp = new ShipComponent(index, "Plasma cannon Mk2", 6, 4,
```

Comment:
Was there a reason to change production and metal cost of Plasma cannons?

Reply:
I just thought that will increase the difficult 🙄